Disable broken defines: pkg-config likely returns more than single library to link, so it cannot
be passed as single argument to dlopen
--- Vulkan-Tools-vulkan-sdk-1.4.321.0/cube/CMakeLists.txt.orig	2025-07-07 18:34:37.000000000 +0200
+++ Vulkan-Tools-vulkan-sdk-1.4.321.0/cube/CMakeLists.txt	2025-07-23 21:16:01.334182535 +0200
@@ -307,16 +307,6 @@ if (ANDROID)
     return()
 endif()
 
-if (XCB_LINK_LIBRARIES)
-    target_compile_definitions(vkcube PRIVATE "XCB_LIBRARY=\"${XCB_LINK_LIBRARIES}\"")
-endif()
-if (X11_LINK_LIBRARIES)
-    target_compile_definitions(vkcube PRIVATE "XLIB_LIBRARY=\"${X11_LINK_LIBRARIES}\"")
-endif()
-if (WAYLAND_CLIENT_LINK_LIBRARIES)
-    target_compile_definitions(vkcube PRIVATE "WAYLAND_LIBRARY=\"${WAYLAND_CLIENT_LINK_LIBRARIES}\"")
-endif()
-
 # ----------------------------------------------------------------------------
 # vkcubepp
 
@@ -360,16 +350,6 @@ target_include_directories(vkcubepp PRIV
 target_compile_definitions(vkcubepp PRIVATE ${ENABLED_CUBE_PLATFORMS})
 target_link_libraries(vkcubepp ${CMAKE_DL_LIBS} Vulkan::Headers)
 
-if (XCB_LINK_LIBRARIES )
-    target_compile_definitions(vkcubepp PUBLIC "XCB_LIBRARY=\"${XCB_LINK_LIBRARIES}\"")
-endif()
-if (X11_LINK_LIBRARIES)
-    target_compile_definitions(vkcubepp PUBLIC "XLIB_LIBRARY=\"${X11_LINK_LIBRARIES}\"")
-endif()
-if (WAYLAND_CLIENT_LINK_LIBRARIES)
-    target_compile_definitions(vkcubepp PUBLIC "WAYLAND_LIBRARY=\"${WAYLAND_CLIENT_LINK_LIBRARIES}\"")
-endif()
-
 if(APPLE)
     install(
         TARGETS vkcubepp
